client/styles/components/_toolbar.scss

```diff
@@ -144,6 +144,6 @@
 .checkbox__autorefresh{
 	cursor: pointer;
 	@include themify(){
-		color:getThemifyVariable('logo-color');
+		accent-color:getThemifyVariable('logo-color');
```

Comment:
Looks good, and it seems like this property has pretty good support: https://caniuse.com/mdn-css_properties_accent-color  It's definitely working for me!

I'm wondering if we want some sort of fallback for older browsers which don't support `accent-color`.  Like will the `color` setting work there or will it do nothing?  I'm reading some articles and it seems like it would do nothing but I'm not sure.  Do you know?  (it's ok if you don't)

Reply:
@lindapaiste See, the color property is not working anywhere because it is not the property to change the color of a input checkbox ( as far as i know ). May be accent color will not work in some older browsers but color property will not work anywhere because it is not the property to change the color of a input checkbox. So its better to use accent color.
I run this code locally and in my browsers it works perfectly file... You can see that in the issue #2808 in expected behaviour.

Reply:
Ok yeah I was wondering if there was an easy way to get backwards compatibility but it seems like there is not.  So this looks good.